src/community/youtube-channel.md

```diff
@@ -0,0 +1,35 @@
+---
+layout: section
+title: 'Beam YouTube channel'
+section_menu: section-menu/community.html
+permalink: /community/youtube-channel/
+---
+
+# Beam YouTube channel
+## What is it and what are the goals?
+The goal of the [Beam YouTube channel](https://www.youtube.com/channel/UChNnb_YO_7B0HlW6FhAXZZQ) is have a central repository with all the videos on or related to the Apache Beam project.
+This can range from very tech-focused talks, to more business- and use case-focused talks; talks delivered at meet ups, conferences or on a webcast.
+
+People who have a recording related to Apache Beam can share this on the channel. The process is documented below.
+
+The YouTube channel found here is owned by the Apache Beam PMC.
+
+<iframe width="560" height="340" src="http://www.youtube.com/embed/bhj4Bjczb8I" frameborder="0" allowfullscreen></iframe>
+
+## Process to add videos to the Apache Beam YouTube channel
+
+When recording a session, make sure you have the following:
+- Get a verbal/written agreement of the people talking, allowing you to record and share the session.
+- Get a verbal/written agreement of the premises where you are filming to record and share the session.
+- Get a verbal/written agreement of the person recording the video that you can share the recording afterwards.
+- Check with local laws if any extra agreements need to be in place to allow to record and share the session.
+
+When you want to add your recording to the channel, do the following:
+- Send an e-mail to the @dev mailing list with:
```

Comment:
Send email to `[dev@beam.apache.org]({{ site.baseurl }}/community/contact-us)` with: